vhost-user-backend/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "vhost-user-backend"
-version = "0.18.0"
+version = "0.19.0"
 authors = ["The Cloud Hypervisor Authors"]
```

Comment:
@stefano-garzarella is it ok to add a feature and create a release i the same PR?

Reply:
nope, but I think this come up from merging `main` in this branch, indeed I asked to rebase it on main instead of merging, otherwise review it's a mess